### PR TITLE
Minor tweaks for performance optimization

### PR DESCRIPTION
**Pull request by munwaikong**
This PR contains a couple of renaming typos and variables as well as 2 minor tweaks for performance improvements:

1) Reduce the number of calls to `gl.getUniformLocation()` as per recommendation of [mdn](https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/WebGL_best_practices)

2) Reduce the number of calls to `clearTexture` every RAF if the state is the same as before.

### CODE REVIEW

**Comment by PTaylour** [comment]
@munwaikong fantastic, thank you. I'll review today.

File: src/ProcessingNodes/processingnode.js [hunk 1]
```diff
@@ -98,12 +96,13 @@ class ProcessingNode extends GraphNode {
 
         //calculate texutre units for input textures
```
Comment (PTaylour) [inline src/ProcessingNodes/processingnode.js]:
```suggestion
        // calculate texture units for input textures
```

**Comment by munwaikong** [comment]
In terms of benchmarking, what I did was comparing the CPU, frames per second and timings reported in Chrome Dev Tools Performance tab.

Find some screenshots below:

Snapshot of original performance (without changes applied)
m

Snapshot of performance with changes applied


If you flip between the 2 images, you can see a lower CPU usage and the script times are slightly lower as well.

**Comment by PTaylour** [comment]
Awesome, thanks for sharing those :)

**Pull request merged**